spk/sabnzbd/Makefile

```diff
@@ -36,16 +36,14 @@ ADMIN_PORT = $(SERVICE_PORT)
 
 POST_STRIP_TARGET = sabnzbd_extra_install
 
-BUSYBOX_CONFIG = nice
-# Add ionice for ARCHs that support it
-ifeq ($(findstring $(ARCH),88f6281 qoriq),$(ARCH))
-BUSYBOX_CONFIG += ionice
-endif
-ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
+# Add nice and ionice commands
+ENV += BUSYBOX_CONFIG="nice ionice"
```

Comment:
I recommend to include these commands in Python 3 busybox so that it can be reused instead of rebuilt here.